### ISSUE TITLE
Toggleable "/" behavior

### ISSUE THREAD

**Issue by aengelberg** [Matthias247/jawampa#18]
I have a (non-jawampa) WAMP client that's trying to connect to my jawampa server at the root path ( "/" ) and is expecting a websocket handshake. I tried to configure my SimpleWampWebsocketListener to listen on the root path (by putting "/" as the uri) but it is still serving a 200 OK response "This server provides a wamp router on path" instead of the websocket handshake. Thus, the client returns the error message "Error during WebSocket handshake: Unexpected response code: 200"

Is there a way to toggle the behavior that sets up that handler, so that I can set up my WAMP router to listen on the root path, without this placeholder getting in the way?

**Comment by Matthias247** [comment]
Hi, I just tried to reproduce this. I set up the SimpleListener with

    URI serverUri = URI.create("ws://0.0.0.0:8080/");
    SimpleWampWebsocketListener server = new SimpleWampWebsocketListener(router, serverUri, null);

and can connect just fine with a jawampa client and a javascript (autobahn) client using the URL `"ws://127.0.0.1:8080/"`.
From the server side code I also can't see anything that could cause the behavior that you see. Because the WAMP upgrade path is checked first. And only if the request is no upgrade request the request will be forwarded and treated as a normal HTTP request. Maybe your client library does not support the root path?
